### PR TITLE
User policy not set by default

### PR DESCRIPTION
**Pull request by Bento007**
* Creating an object does not automatically create and attach a policy. The statement setter will create a policy and attach it to cloud node if none exists. This will prevent an excessive number of dummies policies from being created whenever a user is created.
* A user attached policy is not created by default since most user will not have a policy directly attached.

### CODE REVIEW

File: fusillade/clouddirectory.py [hunk 1]
```diff
@@ -650,14 +675,20 @@ def statement(self):
...
+    def _set_statement(self, statement: str):
         params = [
             UpdateObjectParams('IAMPolicy',
                                'Statement',
                                ValueTypes.StringValue,
                                statement,
                                UpdateActions.CREATE_OR_UPDATE)
         ]
-        self.cd.update_object_attribute(self.policy, params)
+        if not self.policy:
```
Comment (DailyDreaming) [inline fusillade/clouddirectory.py]: Should this be:
```
        if not self.policy:
            self.create_policy(statement)
        self.cd.update_object_attribute(self.policy, params)
```
?

File: fusillade/config.py [hunk 2]
```diff
@@ -12,6 +12,7 @@ class Config:
     def get_admin_emails(cls):
         if not cls._admin_emails:
             cls._admin_emails = [admin.strip() for admin in os.environ['FUS_ADMIN_EMAILS'].split(',') if admin.strip()]
+            assert cls._admin_emails, "initial administrator must be specified."
```
Comment (DailyDreaming) [inline fusillade/config.py]: Could the message contain info about setting `FUS_ADMIN_EMAILS`?

File: fusillade/clouddirectory.py (hunk 1 above)
Comment (Bento007) [reply fusillade/clouddirectory.py]: no but it should be 
```
        if not self.policy:
            self.create_policy(statement)
        else:
            params = [
                UpdateObjectParams('IAMPolicy',
                                   'Statement',
                                   ValueTypes.StringValue,
                                   statement,
                                   UpdateActions.CREATE_OR_UPDATE)
            ]
            self.cd.update_object_attribute(self.policy, params)
```

Fixed

File: fusillade/config.py (hunk 2 above)
Comment (Bento007) [reply fusillade/config.py]: Fixed

**Pull request merged**